theseus/geometry/se3.py

```diff
@@ -171,10 +181,12 @@ def _log_map_impl(
         return SE3_base.log(self.tensor, jacobians=jacobians)
 
     def _compose_impl(self, se3_2: LieGroup) -> "SE3":
-        return SE3(tensor=SE3_base.compose(self.tensor, se3_2.tensor))
+        return SE3(
+            tensor=SE3_base.compose(self.tensor, se3_2.tensor), disable_checks=True
```

Comment:
Maybe `disable_checks=False` is better since composition (matrix multiplication) might accumulate numerical errors.

Reply:
Changed by `strict_checks=False` so they are normalized if the check fails.